apps/react-workshop/src/DropdownMenu.stories.tsx

```diff
@@ -336,3 +339,60 @@ WithContent.decorators = [
     </div>
   ),
 ] satisfies StoryDecorator[];
+
+export const HideMenuWhenTriggerHidden = () => {
```

Comment:
I feel like this story is unnecessary when we already have docs + e2e test. Why do we need all three?